lib/voice/VoiceConnection.js

```diff
@@ -7,23 +7,12 @@ const Dgram = require("dgram");
 const Piper = require("./Piper");
 const VoiceDataStream = require("./VoiceDataStream");
 let WebSocket = typeof window !== "undefined" ? window.WebSocket : require("ws");
+const {safeRequire} = require("../util/utils");
+WebSocket = safeRequire("uws");
```

Comment:
Same comment as above.
```suggestion
WebSocket = safeRequire("uws") || WebSocket;
```